gossip/c_block_callbacks.go

```diff
@@ -222,22 +245,10 @@ func consensusCallbackBeginBlockFn(
 					// sort events by Lamport time
 					sort.Sort(confirmedEvents)
```

Comment:
I think this sorting needs to be moved up together with the `&inter.Block{ ... }` initialization to have a canonical order of the confirmed events.